### PR TITLE
Added type of care attribute

### PR DESCRIPTION
**Pull request by gescobardev**
## Description
Added `fieldServiceTypeOfCare` attribute to `healthServicesListingPage.graphql.js` so that we can use that attribute to group different health services rather than using `parent` attribute.

Since `fieldServiceTypeOfCare` does not include a `weight` the way `parent` does, we cannot order the type of care by weight. As decided in the meeting with CMS, I hard-coded the groupings. 

In addition I removed `sortServices()` since we are not going to be needing it and can handle the sorting within the liquid template.

## Acceptance criteria
- [ ] FE renders the same with no visual changes
- [ ] verify that `fieldServiceTypeOfCare` is being utilized instead of `parent`

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by gescobardev** [comment]
@mpelzsherman @brangi 
Couple questions:

1. I sorted the health services belonging to a type of care by weight rather by name. This is how it was previously done and I am using [this](https://staging.va.gov/pittsburgh-health-care/health-services/) as a reference. If I were to sort by name, the order would be off a little. Just want to confirm that I am sorting correctly. 
2. Should I add specific id/classes to the h2 and section tags ? There weren't any attached to them to begin with.

**Comment by mpelzsherman** [comment]
Outstanding work, @gescobardev ! Answers to your questions below:

> 1. I sorted the health services belonging to a type of care by weight rather by name. This is how it was previously done and I am using [this](https://staging.va.gov/pittsburgh-health-care/health-services/) as a reference. If I were to sort by name, the order would be off a little. Just want to confirm that I am sorting correctly.

You did the right thing! Sorting within each group by weight works just fine.

> 2. Should I add specific id/classes to the h2 and section tags ? There weren't any attached to them to begin with.

No, I don't think there is any need for that.

File: src/site/stages/build/drupal/metalsmith-drupal.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const {
   createPastEventListPages,
   addGetUpdatesFields,
   addPager,
-  sortServices,
+  // sortServices,
```
Comment (mpelzsherman) [inline src/site/stages/build/drupal/metalsmith-drupal.js:19]: Please remove the comment, thanks!

**Review by mpelzsherman — APPROVED** [review]

**Comment by mpelzsherman** [comment]
Looks like we are running some a11y tests as part of the build. The failures do not seem related to these changes.

Let's work with @1Copenut to see how to resolve.

File: src/site/stages/build/drupal/metalsmith-drupal.js (hunk 1 above)
Comment (gescobardev) [reply src/site/stages/build/drupal/metalsmith-drupal.js:19]: Thank you for catching that.

**Comment by gescobardev** [comment]
@department-of-veterans-affairs/vsa-public-websites-frontend or @department-of-veterans-affairs/vsa-caregiver-frontend 
Whenever you get a chance, can you review this?
Thank you in advance.

File: src/site/layouts/health_services_listing.drupal.liquid [hunk 2]
```diff
@@ -42,18 +42,30 @@
...
+          %}
+
+          {% include "src/site/includes/health_services_listing_services.liquid" with
+            clinicalHealthServices
+            typeOfCare = "Social programs and services"
+          %}
+
+          {% include "src/site/includes/health_services_listing_services.liquid" with
+            clinicalHealthServices
+            typeOfCare = "Other services"
```
Comment (kelsonic) [inline src/site/layouts/health_services_listing.drupal.liquid:67]: Does this assign both variables to "Other services"? Pretty cool syntax, have not seen this before 🙂

**Review by kelsonic — APPROVED** [review]
lgtm! 🎉

File: src/site/layouts/health_services_listing.drupal.liquid (hunk 2 above)
Comment (mpelzsherman) [reply src/site/layouts/health_services_listing.drupal.liquid:67]: Actually it looks like `clinicalHealthServices` gets passed in directly.

Comment (gescobardev) [reply src/site/layouts/health_services_listing.drupal.liquid:67]: Oh no, just `typeOfCare` is assigned to "Other services" 
`clinicalHealthServices` is an object containing individual health services data.
That would've been cool tho 🥶

**Pull request merged**